spark-jobs/src/main/scala/filodb/downsampler/DownsamplerSettings.scala

```diff
@@ -16,6 +16,8 @@ import filodb.core.store.{IngestionConfig, StoreConfig}
   */
 object DownsamplerSettings extends StrictLogging {
 
+  Kamon.init()
```

Comment:
I worry that this may not be the "first thing" loaded by spark executor. Lets ensure that metrics work on the spark jobs as well.